mentors/csharp/n.json

```
@@ -1,5 +1,13 @@
 [
   {
+    "github_username": "nahuel-ianni",
+    "name": "Nahuel Ianni",
+    "link_text": null,
+    "link_url": null,
+    "avatar_url": "https://avatars1.githubusercontent.com/u/8545295",
+    "bio": "I've been working as a consultant on a variety of industries and roles since 2010, mostly focused on software engineering in .NET environments"
+  }
+  ,{
```

```suggestion
  },
  {
```
Just a minor formatting change.